src/ServiceInsight/Shell/About.xaml

```diff
@@ -48,14 +48,14 @@
 
                 <TextBlock Style="{StaticResource InfoText}">
                     <Run Text="Version  " />
-                    <Run Text="{Binding AppVersion}" />
+                    <Run Text="{Binding AppVersion, Mode=OneWay}" />
                     <Run Text="  Commit hash:  " />
-                    <Run Text="{Binding CommitHash}" />
+                    <Run Text="{Binding CommitHash, Mode=OneWay}" />
                 </TextBlock>
 
                 <TextBlock Style="{StaticResource InfoText}" Visibility="{Binding IsSplash, Converter={StaticResource BoolToVisibilityConverterInverted}}">
```

Comment:
So the default value is "Default" which means it uses the value from the dependency property which varies between the controls. So some controls default to one-way (usually read-only controls like TextBlock) and some use two-way binding (like editable controls such as TextBox). 

What is a strange default in fact, is that Run.Text property uses Two-Way as default and that *could* be because it is used in other controls such as RichTextBox which supports two-way binding.